Test: `FP8ConversionTests.test_fp8_to_unsigned_integers` is flaky

As per offline discussion with @gmarkall we'd skip it unconditionally for now. Though this seems to only happen on Windows. Example: https://github.com/NVIDIA/numba-cuda/actions/runs/21833980445/job/63001311747
```
______________ FP8ConversionTests.test_fp8_to_unsigned_integers _______________
[gw5] win32 -- Python 3.12.10 C:\actions-runner\_work\_tool\Python\3.12.10\x64\python.exe

self = <numba.cuda.tests.cudapy.test_fp8_bindings.FP8ConversionTests testMethod=test_fp8_to_unsigned_integers>

    def test_fp8_to_unsigned_integers(self):
        """Test FP8 conversion to unsigned integer types."""
    
        # Test data: (fp8_type, test_value, expected)
        test_cases = [
            (fp8_e5m2, 10.0, 10),
            (fp8_e4m3, 12.0, 12),
            (fp8_e8m0, 16.0, 16),
        ]
    
        for fp8_type, test_val, expected in test_cases:
            with self.subTest(fp8_type=fp8_type.__name__, value=test_val):
    
                @cuda.jit
                def kernel(result):
                    fp8_val = fp8_type(float32(test_val))
    
                    result[0] = uint64(uint8(fp8_val))
                    result[1] = uint64(uint16(fp8_val))
                    result[2] = uint64(uint32(fp8_val))
                    result[3] = uint64(fp8_val)
    
                result = np.zeros(4, dtype=np.uint64)
                kernel[1, 1](result)
    
>               self.assertTrue(np.all(result == expected))
E               AssertionError: np.False_ is not true
```

--- a/numba_cuda/numba/cuda/_internal/cuda_fp8.py
+++ b/numba_cuda/numba/cuda/_internal/cuda_fp8.py
@@ -760,8 +760,8 @@ _from___nv_fp8_e5m2_to_uint32_lower(shim_stream, shim_obj)
 def _from___nv_fp8_e5m2_to_uint64_lower(shim_stream, shim_obj):
     shim_raw_str = """
     extern "C" __device__ int
-    _ZNK13__nv_fp8_e5m2cvmEv_nbst(unsigned long &retval, __nv_fp8_e5m2 *self) {
-        retval = self->operator unsigned long();
+    _ZNK13__nv_fp8_e5m2cvyEv_nbst(unsigned long long &retval, __nv_fp8_e5m2 *self) {
+        retval = self->operator unsigned long long();
         return 0;
     }
         """
@@ -770,7 +770,7 @@ def _from___nv_fp8_e5m2_to_uint64_lower(shim_stream, shim_obj):
     def impl(context, builder, fromty, toty, value):
         context.active_code_library.add_linking_file(shim_obj)
         callconv = FunctionCallConv(
-            itanium_mangled_name="_ZNK13__nv_fp8_e5m2cvmEv",
+            itanium_mangled_name="_ZNK13__nv_fp8_e5m2cvyEv",
             shim_writer=shim_writer,
             shim_code=shim_raw_str,
         )

--- a/numba_cuda/numba/cuda/tests/cudapy/test_fp8_bindings.py
+++ b/numba_cuda/numba/cuda/tests/cudapy/test_fp8_bindings.py
@@ -218,7 +218,6 @@ class FP8ConversionTests(CUDATestCase):
                         )
                     )
 
-    @unittest.skip("flaky test - needs investigation (NVIDIA/numba-cuda#819)")
     def test_fp8_to_unsigned_integers(self):
         """Test FP8 conversion to unsigned integer types."""
 

